lib/utils/usage-utils.ts

```diff
@@ -8,6 +8,6 @@ import { StackProps } from "@aws-cdk/core";
  */
  export function withUsageTracking(usageIdentifier: string, stackProps?: StackProps): StackProps {
     const result =  stackProps ?? {};
-    const trackableDescription = `${result.description?? ""} SSP tracking (${usageIdentifier})`.trimLeft();
+    const trackableDescription = `${result.description?? ""} Blueprints tracking (${usageIdentifier})`.trimLeft();
```

Comment:
Verify if the tracking code needs updating

Reply:
it is good, they look for qs* anywhere in the description. real clear and unambiguous approach :))